modules/qna/src/backend/providers/nlu.ts

```diff
@@ -125,6 +129,22 @@ export default class Storage implements QnaStorage {
     return ids
   }
 
+  private async checkForDuplicatedQuestions(newQuestions, editingQnaId?) {
+    let allQuestions = await this.fetchAllQuestions()
+
+    if (editingQnaId) {
+      // when updating, we remove the question from the check
+      allQuestions = allQuestions.filter(q => q.id !== editingQnaId)
+    }
+
+    const questionsList = _.flatten(_.map(allQuestions, entry => entry.data.questions))
```

Comment:
just so you know _.flatten + _.map = _.flatmap

Reply:
good to know ! thx